pipelines/h3c/rtp/v1/task_scripts/do_ADD_LIBRARIAN_SSINS.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+set -e
+
+# import common functions
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# Parameters are set in the configuration file. Here we define their positions,
+# which must be consisten wtih the config.
+# 1 - (raw) filename
+fn=$(inject_hh ${1})
+
+# get the integer portion of the JD
+jd=$(get_int_jd ${1})
+
+midfix="_SSINS_"
+prefix="${fn%.*}"${midfix}
+
+for suffix in data.h5 z_score.h5 mask.h5 flags.h5 match_events.yml; do
+  file=${prefix}${suffix}
+  echo upload_to_librarian.py local-rtp $file ${jd}/$file
```

Comment:
The command being `echo`'d here does not match the one actually being run.

Reply:
I copied `do_ADD_LIBRARIAN_ANT_METRICS.sh` on this one and this is also the syntax in `do_UPLOAD_HH.sh`. I do not know whether the `echo`, the command, or neither option is the right style to use, but I absolutely agree they should match.

Reply:
If this is what is in the other scripts, then those should be changed as well. Basically, what is `echo`'d in this line should match the command that is actually run. The `do_SSINS.sh` script is right in this respect.

Reply:
Woops, good catch. I think when the API to the librarian changed, I forgot to update the echo line. I'll start a separate PR for that. let's just get it right in this script for this PR.

Reply:
I believe this is done correctly now.

Reply:
Yep, looks great thanks!